tools/manually-handling-sharding-ddl-locks.md

```diff
@@ -0,0 +1,415 @@
+- `exec`: flag; boolean; `--exec`; optional; cannot be specified simultaneously with the `--skip` parameter; if being specified, ask the DM-worker to execute the corresponding DDL of the lock 
+- `skip`: flag; boolean; `--skip`; optional; cannot be specified simultaneously with the `--exec` parameter; if being specified, ask the DM-worker to skip the corresponding DDL of the lock 
+- `task-name`: non-flag; string; not optional; specify the name of the task containing the lock that is going to execute the breaking operation (you can check if a task contains the lock via [query-status](../task-handling/query-status.md))
+
+##### Example of results
+
+```bash
+» break-ddl-lock -w 127.0.0.1:8262 --exec test
+{
+    "result": true,                                        # check if the lock breaking operation succeeds
+    "msg": "",                                             # check the reason why the lock breaking operation fails
+    "workers": [                                           # the list of DM-workers which executes the lock breaking operation (currently you can only break the lock on one DM-worker at a single operation)
+        {
+            "result": false,                               # check if the lock breaking operation of the DM-worker succeeds
+            "worker": "127.0.0.1:8262",                    # the address of the DM-worker (DM-worker ID)
+            "msg": ""                                      # check the reason why the lock breaking operation of the DM-worker fails
+        }
+    ]
+}
+```
+
+### Support Scenarios
+
+Currently, using the command `unlock-ddl-lock` or `break-ddl-lock` can only handle the sharding DDL lock under the following abnormal scenarios. You should consult the related developers if you encounter other anomalies.
+
+#### Some DM-workers go offline
+
+##### The reason for the abnormal lock
+
+Before `DM-master` tries to automatically unlock the sharding DDL lock, all the DM-workers need to receive the sharding DDL event (for details, see [shard merge principles](./shard-merge.md#Principles)). If the sharding DDL event is already in the synchronization process, and some DM-workers have gone offline and are not to be restarted (these DM-workers have been removed according to the application demand), then the sharding DDL lock cannot be automatically synchronized and unlocked because not all the DM-workers can receive the DDL event.
+
+> If you do not need to make some DM-workers offline in the process of synchronizing sharding DDL events, a better solution is to use `stop-task` to stop the running tasks first, make the DM-workers go offline, remove the corresponding configuration information from the task configuration file, and finally use `start-task` and the new task configuration to restart the synchronization task. 
```

Comment:
`do not need to` => `need to`?

Reply:
Yes, it should be: "If you need to make some DM-workers offline when not in the process of synchronizing sharding DDL events".